### PR TITLE
oauthlib: update to v 3.2.2

### PR DESCRIPTION
**Pull request by akabanovs**
- sha256 updated
 - about section updated

### CODE REVIEW

**Review by ELundby45 — COMMENTED** [review]
I suggest changing `- blinker >=1.4.0` to using the exact version `1.4`

https://github.com/oauthlib/oauthlib/blob/v3.2.2/requirements.txt#L2

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,7 +15,6 @@ source:
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:11]: fn: {{ name }}-{{ version }}.{{ compress_type }}  - this can be removed, I think

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {% set name = "oauthlib" %}
-{% set version = "3.2.1" %}
+{% set version = "3.2.2" %}
 {% set compress_type = "tar.gz" %}
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: all those extra variables used only once, isn't that is over engineered, can we simplify it and remove compress_type, hash_type and make consistent with other feedstocks ?

Comment (varlackc) [reply recipe/meta.yaml]: It is ok to simplify the recipe if the variables are only used once.

**Comment by akabanovs** [comment]
@ELundby45  I used the range >=1.4 based on the setup.py file, see

https://github.com/oauthlib/oauthlib/blob/v3.2.2/setup.py#L21

Please let me know if I am wrong, but I believe that the setup.py file is the source of truth here because requirements.txt file, which is sometimes read by setup.py, is not read in this case. In fact, the requirements.txt file is not included in the files when I download and extract the source files from pypi.

**Review by ELundby45 — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (akabanovs) [reply recipe/meta.yaml:11]: yes, done that. Did not notice initially.

File: recipe/meta.yaml (hunk 2 above)
Comment (akabanovs) [reply recipe/meta.yaml]: Done

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**